invenio_pages/templates/invenio_pages/administration/page_details.html

```diff
@@ -0,0 +1,9 @@
+{#
+  Copyright (C) 2022 CERN.
+
+  Invenio Pages is free software; you can redistribute it and/or modify it
+  under the terms of the MIT License; see LICENSE file for more details.
+#}
+
+{% extends "invenio_administration/details.html" %}
```

Comment:
It's not necessary to override the `invenio-administration` details page if we don't need to change/add anything. It will automatically use the `invenio_administration/details.html` file if we don't specify any `template` in config file. Same for the search page.

Reply:
In my case I'm needing to override the invenio-administration details and search views, as I need to make adjustments to the jsx for it to look nice.